Logger/FileLogger.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+using System.IO;
+
+
+namespace Logger;
+
+public class FileLogger(string file) : BaseLogger
+{
+    private string _file { get; set; } = file;
```

Comment:
Naming convention for Properties is PascalCase.
```suggestion
    private string File { get; set; } = file;
```